grakn-core/src/main/java/ai/grakn/GraknTx.java

```diff
@@ -320,6 +320,7 @@
      * @return true if the current transaction is read only
      */
     @CheckReturnValue
+    @API
     boolean isReadOnly();
```

Comment:
Since we have an opportunity to define our API now, how about a method that returns the `GraknTxType` instead of this more-specific one?

Reply:
Good catch! Will do sir!